.github/workflows/snapshot.yml

```
@@ -0,0 +1,28 @@
+name: Snapshot
+
+on: workflow_dispatch
+
+jobs:
+  publish:
+    name: Publish snapshot version
+    runs-on: ubuntu-latest
+    env:
+      CI: true
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@main
+        with:
+          # This makes Actions fetch all Git history so that Changesets can generate changelogs with the correct commits
+          fetch-depth: 0
+          token: ${{ secrets.SEEK_OSS_CI_GITHUB_TOKEN }}
+
+      - name: Install Dependencies
```

Still have to setup node yeah?
```suggestion
      - name: Setup Node.js 14.x
        uses: actions/setup-node@main
        with:
          node-version: 14.x

      - name: Install Dependencies
```

---

Oh yeah, must've ovezealously deleted when removing pnpm from the vanilla copy